### ISSUE TITLE
Lispkit gtkalbel / container error on Debian 8.

### ISSUE THREAD

**Issue by aykaramba** [AeroNotix/lispkit#87]
Hi.

* I managed to compile Lispkit on Debian 8.  On Debian 7 asdf is too old apparently, which is fine.

* I cannot compile Lispkit with libwebkit2gtk-4.0-37 ... this is fine.

* I can compile Lispkit with libwebkit2gtk-3.0-25, however, on ./lispkit I get the following error: 

gtk-warning **: attempting to add a widget with type gtklabel to a container of type gtkbox, but the widget is already inside a container of type gtkbox, please use gtk_widget-reparent()

Basically Lispkit window starts up but the rendering inside the window hangs and renders the browser unusable.

If this issue requires further testing, please let me know.  I will be watching this ticket.

**Comment by AeroNotix** [comment]
I'll test the setup you mentioned and get back to you. If someone else has this setup immediately available, please feel free to confirm.

**Comment by aykaramba** [comment]
Great!  Let me know if you need me to do any testing as well.  I would be happy to provide detailed feedback.

**Comment by ahungry** [comment]
Sounds like the render issue I had on a ticket a year or so ago

**Comment by aykaramba** [comment]
I reinstalled a laptop with debian 8, all standard libraries and such, made sure libwebkit2gtk-3.0-25 and now Lispkit is rendering.

I think ahungry is correct.

So, on this environment, Lispkit builds fine and renders fine.  However the clie still shows the error above.

I don't know if it is appropriate to close this ticket or to leave it open because of the error message ... the error doesn't sound like it is too interesting.

AeroNotix, please feel free to close this ticket if you deem it appropriate.  Right now I can get Liskpkit working on a laptop so I can start testing.

Many thanks!

**Comment by AeroNotix** [comment]
Closing.
